music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -210,6 +230,14 @@ async def get_config_entries(
             depends_on=CONF_USE_EXTERNAL_SERVER,
             category="advanced" if local_snapserver_present else "generic",
         ),
+        ConfigEntry(
+            key=CONF_STREAM_IDLE_THRESHOLD,
+            type=ConfigEntryType.INTEGER,
+            default_value=DEFAULT_SNAPSTREAM_IDLE_THRESHOLD,
+            label="Snapcast idle threshold stream parameter",
+            required=True,
+            category="Stream Settings",
```

Comment:
same, category is not meant to be a freetext label

Reply:
this seems to be some super advanced setting, why not mark the category as "advanced" ?

Reply:
> this seems to be some super advanced setting, why not mark the category as "advanced" ?

This value is the key to solve the audio cut-out problem, so it might still need some tweaking. But putting it in Advanced makes sense as it's not something you want to adjust every now and then.

Reply:
I move it to advance